pkg/operator/ceph/controller/controller_utils.go

```diff
@@ -116,6 +118,21 @@ func LoopDevicesAllowed() bool {
 	return loopDevicesAllowed
 }
 
+func SetEnforceHostNetwork(data map[string]string) {
+	Strval := k8sutil.GetValue(data, EnforceHostNetworkSettingName, EnforceHostNetworkDefaultValue)
```

Comment:
nit: local vars should start with lower case
```suggestion
	strval := k8sutil.GetValue(data, EnforceHostNetworkSettingName, EnforceHostNetworkDefaultValue)
```

Reply:
planning on this change?